src/dashboard/src/media/js/ingest.js

```diff
@@ -331,13 +331,6 @@ $(function()
             window.location.href = '/ingest/' + this.model.sip.get('uuid') + '/upload/atk/';
           }
 
-          // redirect to object/resource mapping pages
```

Comment:
Yes.  Since this PR adds automating the matching GUI (for ArchivesSpace), I removed starting up the matching GUI in JS. I'd rather it checked if it was necessary first, but that's non-trivial.  Open to discussion on that one.